src/APBFramelessWindow/NativeTranslucentFramelessWindow.cpp

```diff
@@ -146,7 +142,6 @@ bool NativeTranslucentFramelessWindow::isBottomLeftCornerHit(const QRect &iRect,
 
 bool NativeTranslucentFramelessWindow::isTitleBarHit(const QRect &iRect, const long iBorderWidth, long iX, long iY) {
 	bool isHeightSatisfied = iY >= iRect.top() && iY < iRect.top() + this->titleBar->height();
-	bool isWidthSatisfied = iX >= iRect.left() && iX < iRect.right() - this->titleBar->buttonsWidth();
-
```

Comment:
Clean code principle: `Use white space to associate related things and disassociate weakly related.`